packages/utils/src/geometry.ts

```diff
@@ -247,3 +247,21 @@ export function divide(region: Region, size: RectangleSize, padding: {top?: numb
   }
   return subRegions
 }
+export function withPadding(region: Region, padding: any): Region {
```

Comment:
Don't use `any` if it is possible
```suggestion
export function withPadding(region: Region, padding: number | {top?: number; right?: number; bottom?: number; left?: number}): Region {
```

Reply:
When you will use a padding object without knowing if all of the fields are specified you can use `?? 0` to replace it with 0 if it wasn't specified